lxd/daemon.go

```diff
@@ -754,6 +754,9 @@ func (d *Daemon) createCmd(restAPI *mux.Router, version string, c APIEndpoint) {
 			return
 		}
 
+		// Generate an OpenFGA request ID
```

Comment:
Can you update the comment here please?
Suggestion `Set OpenFGA cache in request context to improve driver performance`

Reply:
fixed